components/assignmentList.js

```diff
@@ -70,16 +70,15 @@ const AssignmentsList = ({
   // Group the assigned drills by date
   const groupedData = useMemo(() => {
     return assigned_data.reduce((acc, curr) => {
-      //round the assigned time to the nearest day
-      const roundedTime = Math.floor(curr.assignedTime / 86400000) * 86400000;
+      const time = getTimezoneOffsetTime(curr.assignedTime);
```

Comment:
I think timezone conversion should be saved until this data is displayed.

Reply:
Also, "today" is not converted to the current timezone but time here is so that comparison will be wack. Put the timezoneoffset functionality inside getFormat.